core/Db/Schema/Mysql.php

```diff
@@ -52,6 +53,14 @@ public function getTablesCreateSql()
                           ) ENGINE=$engine DEFAULT CHARSET=utf8
             ",
 
+            'twofactor_recovery_code'    => "CREATE TABLE {$prefixTables}twofactor_recovery_code (
+                          idrecoverycode BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+                          login VARCHAR(100) NOT NULL,
+                          recovery_code VARCHAR(40) NOT NULL,
```

Comment:
That one differs to the update script, it only creates a `VARCHAR(20)`

Reply:
👍  will fix